lib/bootsy/activerecord/image_gallery.rb

```diff
@@ -10,8 +10,13 @@ module Bootsy
   # that do not point to resources older than the given time
   # limit.
   class ImageGallery < ActiveRecord::Base
-    belongs_to :bootsy_resource, polymorphic: true, autosave: false,
-                                 optional: true
+    if Rails::VERSION::STRING.split(".").first.to_i >= 5
```

Comment:
Hmm.. I'm not sure I'd ditch support for that just yet. Rails 3 projects are still active, much less Rails 4. Rails 5 is still extremely fresh as far as release cycles are concerned, you're probably cutting out a big chunk of people if not supporting < 5. I'd wager to say 90% of the projects I do are on Rails 4 systems.